x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -21,6 +21,8 @@ import type {
 import type { CspServerPluginStart, CspServerPluginStartDeps } from '../../../types';
 import { CLOUD_SECURITY_POSTURE_PACKAGE_NAME } from '../../../../common/constants';
 
+const AGENTLESS_POLICY_ID = 'agentless'; // the policy id defined here: https://github.com/elastic/project-controller/blob/main/internal/project/security/security_kibana_config.go#L86
```

Comment:
We already have the policy id const defined in `x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts` , let's expose it via fleet exports and use it here? This way we make sure we update it in one place and the rest will be updated too. 
And it's a good idea to mention where the policy id is defined in the project-controller. If you go ahead with using the const from fleet, let's add this comment there instead for one source of truth

Reply:
Good idea, done.